src/lockstep/lockstep_api.py

```diff
@@ -156,7 +160,15 @@ def send_request(self, method: str, path: str, body: object, query_params: objec
             headers["Api-Key"] = self.apiKey
         elif self.bearerToken:
             headers["Authorization"] = "Bearer " + self.bearerToken
-    
+
         response = requests.request(method, url, headers=headers)
-        return response.json()
+
+        execution_time = time.time() - start_time
+        result = response.json()
+
+        # times are represented in milliseconds
+        result["RoundTripTime"] = execution_time * 1000
+        result["ServerDuration"] = response.elapsed.microseconds / 1000
```

Comment:
Close!  I think this is going to be `response.headers["ServerDuration"]` or something similar.

Reply:
Fixed! Thanks for the hint :)